### PR TITLE
Added more cargo shuttle names

### PR DESCRIPTION
**Pull request by Starbuckss14**
## About the PR
Added more cargo shuttle names

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Cargo shuttle names

### CODE REVIEW

File: Resources/Prototypes/Datasets/Names/cargo_shuttle.yml [hunk 1]
```diff
@@ -9,3 +9,7 @@
   - Shuttle McShuttleface
   - ShuttleShuttle
   - Urist McShuttle
+  - Clown Town
```
Comment (lzk228) [inline Resources/Prototypes/Datasets/Names/cargo_shuttle.yml:12]: doesn't feel like for cargo shuttle

**Review by lzk228 — COMMENTED** [review]
also doesn't require changelog

**Comment by IProduceWidgets** [comment]
Do these actually get used? I've never seen them in game before.

**Comment by lzk228** [comment]
> Do these actually get used? I've never seen them in game before.

they supposed to be used for the cargo shuttle's name, but the dataset is really never used 
image

**Comment by Starbuckss14** [comment]
> > Do these actually get used? I've never seen them in game before.
> 
> they supposed to be used for the cargo shuttle's name, but the dataset is really never used image

I believe they show on the shuttle console and the IFF mass scanner.

**Comment by deltanedas** [comment]
this is only used on mining station and other ancient forks

**Comment by slarticodefast** [comment]
Yeah, this dataset isn't used anywhere, so the PR does nothing right now. If you want to you could maybe add the shuttle name to the IFF radar screen or as a label on the shuttle console UI.

**Comment by Starbuckss14** [comment]
> Yeah, this dataset isn't used anywhere, so the PR does nothing right now. If you want to you could maybe add the shuttle name to the IFF radar screen or as a label on the shuttle console UI.

understood if this ever changes I will retry

**Pull request closed without merging**